tests/test-runner/bin/zts-report.py

```diff
@@ -265,6 +265,7 @@
     'rsend/rsend_024_pos': ['FAIL', '5665'],
     'rsend/send-c_volume': ['FAIL', '6087'],
     'snapshot/clone_001_pos': ['FAIL', known_reason],
+    'snapshot/rollback_003_pos': ['FAIL', known_reason],
```

Comment:
Maybe we should link the old issue that was closed by https://github.com/zfsonlinux/zfs/commit/3fd3e56cfd543d7d7a1bf502bfc0db6e24139668 and reopen it: https://github.com/zfsonlinux/zfs/issues/6143

Reply:
Sure. That sounds better than what I’ve done so far. I can do this.

Reply:
@prakashsurya thank you for catching this.